chromium/background.js

```diff
@@ -618,3 +627,38 @@ chrome.runtime.onConnect.addListener(function (port) {
     });
   }
 });
+
+// This is necessary for communication with the popup in Firefox Private
+// Browsing Mode, see https://bugzilla.mozilla.org/show_bug.cgi?id=1329304
+chrome.runtime.onMessage.addListener(function(message, sender, sendResponse){
```

Comment:
Can you confirm that this will only receive messages from background.js, not from content scripts or other extensions?